src/main/java/io/electrum/prepaidutility/api/EventsResource.java

```diff
@@ -0,0 +1,77 @@
+package io.electrum.prepaidutility.api;
+
+import io.electrum.prepaidutility.model.ErrorDetail;
+import io.electrum.prepaidutility.model.PurchaseResponse;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import io.swagger.annotations.Authorization;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.container.AsyncResponse;
+import javax.ws.rs.container.Suspended;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.Request;
+import javax.ws.rs.core.SecurityContext;
+import javax.ws.rs.core.UriInfo;
+
+@Path(PrepaidUtilityApi.API_BASE_PATH)

```

Comment:
I think this is how we end up duplicating the API base path. It's set here but it is also specified in the input to the swagger creation in the POM and we end up duplicating the base path in the description of the operation.
```suggestion
@Path(EventsResource.PATH)
```

Considering this is a fairly common mistake with a real impact on the published and implemented API I wonder if we should create a test for it. If so, we should probably create it in the base API so it can be implemented across our APIs.

Reply:
Was just following the pattern here, it's the same for the other resources in this API. Better to stick with a common feel for all resources or fix the broken pattern for this resource only? I vote for keep the pattern...not sure how we will ever fix this though

Reply:
So we fixed this in other APIs by dropping the duplication from all the resource classes. Technically that's a breaking change in the API (and it's reflected in a breaking change in the swagger) and it's definitely a breaking change within our own server implementations of the APIs. But, rather than actually bump the API major version (which would cause a lot of admin since it changes the service URL) we highlight it quite explicitly in the release notes i.e. that it's a breaking change but the major version doesn't change. You can check [v1.6.0 and v1.5.0 of the SUV API](https://electrumpayments.github.io/suv-service-interface-docs/specification/release-notes/)

Reply:
also, this seems to be causing a nuisance for some customers so we should fix this as suggested if possible rather than kicking the can down the road a little bit further.